MetaMask/metamask-extension
# Fixing some incorrect CSS color vars

## Explanation

**What is the current state of things and why does it need to change?**
There were some incorrect CSS vars being used that weren't rendering the correct colors.

**What is the solution your changes offer and how does it work?**
This PR updates those incorrect CSS var and fixes the hold to reveal button as well as a link on the send page

## Screenshots/Screencaps



### Before


https://user-images.githubusercontent.com/8112138/209023120-5240582b-844e-49b3-be3d-ba4ec849f8f0.mov


Created a story locally and plugged in the text to check this alert visually. Not apart of this PR.

<img width="1213" alt="Screenshot 2022-12-21 at 3 44 54 PM" src="https://user-images.githubusercontent.com/8112138/209024434-39df7dcd-b27a-497a-b01b-03c6f1a66f35.png">


### After


https://user-images.githubusercontent.com/8112138/209023141-b5ef7a2e-b62f-4bca-bc5c-feb94104a01d.mov

Created a story locally and plugged in the text to check this alert visually. Not apart of this PR.
Can't test this without creating a story locally just trust I checked 😉 

<img width="1436" alt="Screenshot 2022-12-21 at 3 43 09 PM" src="https://user-images.githubusercontent.com/8112138/209024512-42165c8d-a615-400c-b181-f67626c70fde.png">

## Manual Testing Steps

- Go to the most recent build of Storybook on this PR
- Search `HoldToRevealButton`
- Click and hold on the button to see the correct colors being used
- Compare with what's currently in `develop` https://metamask.github.io/metamask-storybook/?path=/story/ui-components-app-hold-to-reveal-button-hold-to-reveal-button-stories-js--default-story

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
